llvm/test/CodeGen/RISCV/rvv/fixed-vectors-vp-splice.ll

```diff
@@ -7,10 +7,9 @@
 define <2 x i64> @test_vp_splice_v2i64(<2 x i64> %va, <2 x i64> %vb, i32 zeroext %evla, i32 zeroext %evlb) {
 ; CHECK-LABEL: test_vp_splice_v2i64:
 ; CHECK:       # %bb.0:
-; CHECK-NEXT:    addi a0, a0, -5
-; CHECK-NEXT:    vsetvli zero, a0, e64, m1, ta, ma
-; CHECK-NEXT:    vslidedown.vi v8, v8, 5
 ; CHECK-NEXT:    vsetvli zero, a1, e64, m1, ta, ma
+; CHECK-NEXT:    vslidedown.vi v8, v8, 5
+; CHECK-NEXT:    addi a0, a0, -5
 ; CHECK-NEXT:    vslideup.vx v8, v9, a0
 ; CHECK-NEXT:    ret
 
```

Comment:
Isn't this test (and a bunch other in this file) ill defined?  The specification of this intrinsic says the offset starts the selected region from the concatenation.  Since the concatenation is at most 4 elements long, isn't the result always poison?

Reply:
Yeah I think so. I guess it's because we just decided to use 5 as the offset for all these examples

Reply:
I assuem a copy paste mistake. The immediate for 2xi64 needs to be [-2, 1]. The splice result must include at least one element from the first vector of the concatenation.